yarn.lock

```diff
@@ -2330,14 +2330,22 @@
     apollo-server-types "^3.2.0"
     tslib "^2.4.0"
 
-"@envelop/core@3.0.2", "@envelop/core@^2.4.0":
+"@envelop/core@3.0.2":
   version "3.0.2"
   resolved "https://registry.yarnpkg.com/@envelop/core/-/core-3.0.2.tgz#0c74b7cf2a1b3fc15bef29dfbebea084249bf5a3"
   integrity sha512-0kYBq0gKIARF6vFYw5upUcBgI2oP3wYaDTUfjiX4RhMIkox3DiKPUwRiZHu16ILGr8Qi6p+HTKXRyoSdl9CCqA==
   dependencies:
     "@envelop/types" "3.0.0"
     tslib "2.4.0"
 
+"@envelop/core@^2.4.0":
```

Comment:
This is weird 🤔

Reply:
I think its cause `graphql-armor` still uses `2.4.0` but we do resolutions in monorepo though